website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: "Enabling Continuous Integration (CI)"
-id: "cloud-enabling-continuous-integration"
+title: "dbt Cloud CI job"
```

Comment:
Should we spell out 'continuous integration' in either the title or description to assist in SEO?